rdrf/rdrf/services/io/defs/importer.py

```diff
@@ -902,25 +907,45 @@ def _create_form_permissions(self, registry):
                 form_model.save()
                 logger.info(f"Import groups_readonly for form {form_name}: {groups_readonly}")
 
+    def _create_working_group_types(self):
+        working_group_types = self.data['working_group_types']
+
+        for type_dict in working_group_types:
+            type_model, _ = WorkingGroupType.objects.get_or_create(name=type_dict.get('name'))
+
+            existing_rules = set(type_model.rules.all())
+            imported_rules = set()
+            for rule_dict in type_dict.get('rules', []):
+                rule_model, _ = WorkingGroupTypeRule.objects.update_or_create(type=type_model,
```

Comment:
nitpick: I would start the params on a new line because we're getting pretty indented past 120 chars

```suggestion
                rule_model, _ = WorkingGroupTypeRule.objects.update_or_create(
                    type=type_model,
                    user_group=Group.objects.get(name__iexact=rule_dict.get('user_group')),
                    defaults={'has_default_access': rule_dict.get('has_default_access')}
                )
```